diagnostic.md

```diff
@@ -6,70 +6,71 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+resources :books, only: [:show]
+resources :books, except: [:new, :edit]
```

Comment:
The question was looking for something more like this, 
```
get '/books/:id', to: 'books#show'
get '/books/:id' => 'books#show'
```

But I think you understand the distinction.